test/quic-openssl-docker/run_endpoint.sh

```diff
@@ -69,8 +69,14 @@ if [ "$ROLE" == "client" ]; then
        exit 0
        ;;
     "chacha20")
-       OUTFILE=$(basename $REQUESTS)
-       SSL_CERT_FILE=/certs/ca.pem curl --verbose --tlsv1.3 --tls13-ciphers TLS_CHACHA20_POLY1305_SHA256 --http3 -o /downloads/$OUTFILE $REQUESTS || exit 1
+       for req in $REQUESTS
+       do
+           OUTFILE=$(basename $req)
+           printf "%s " "$OUTFILE" >> ./reqfile.txt
+           HOSTNAME=$(printf "%s\n" "$req" | sed -ne 's,^https://\([^/:]*\).*,\1,p')
+           HOSTPORT=$(printf "%s\n" "$req" | sed -ne 's,^https://[^:/]*:\([^/]*\).*,\1,p')
+       done
```

Comment:
I've approved the PR as-is, but an alternative implementation, that is arguably cleaner, could be:
```suggestion
       reqpaths=()
       for req in $REQUESTS
       do
           reqpaths=("${reqpaths[@]}" "$(basename $req)")
           HOSTNAME=$(printf "%s\n" "$req" | sed -ne 's,^https://\([^/:]*\).*,\1,p')
           HOSTPORT=$(printf "%s\n" "$req" | sed -ne 's,^https://[^:/]*:\([^/]*\).*,\1,p')
       done
       printf "%s\n" "${reqpaths[*]}" > ./reqfile.txt
```
Though (in case it is material), this version appends a newline rather than a space after the last request path.
Yet another  version could redirect the "for" loop, rather than each printf.